org.mixedrealitytoolkit.uxcore/Experimental/List/VirtualizedScrollRectList.cs

```diff
@@ -531,6 +580,27 @@ public void SetItemCount(int newCount)
             visibleValid = false;
             UpdateScrollView(scroll);
         }
+
+        /// <summary>
+        /// Resets the VirtualizedScrollRectList. This may remove items aready
+        /// visible and create new items.
+        /// </summary>
+        public void ResetLayout()
+        {
+            // We only want to reset things if it has already been initialized,
+            // we don't want to initialize it prematurely!
+            if (initialized == false)
+            {
+                return;
+            }
+
+            // We don't want to reset multiple times in one frame.
+            if (resetCalled)
```

Comment:
I don't see where `resetCalled` is set to `true`. Should it be below here, right before the `StartCoroutine`?

Reply:
Thank you for catching that. :D